engine/lib/navigation.php

```diff
@@ -184,16 +184,12 @@ function elgg_register_title_button($handler = null, $name = 'add', $entity_type
 		return;
 	}
 	
-	$href = elgg_generate_url("$name:$entity_type:$entity_subtype", [
+	$href = elgg_generate_url("{$name}:{$entity_type}:{$entity_subtype}", [
 		'guid' => $owner->guid,
 	]);
 	
-	if (empty($href) && isset($handler)) {
```

Comment:
why didn't you remove ``$handler`` from the function params??

Reply:
will do that in a separate commit as it will need to be updated in several views/resources